physics/barycentric_rotating_reference_frame_body.hpp

```diff
@@ -26,15 +26,24 @@ using namespace principia::quantities::_si;
 
 template<typename InertialFrame, typename ThisFrame>
 BarycentricRotatingReferenceFrame<InertialFrame, ThisFrame>::
-BarycentricRotatingReferenceFrame(
-    not_null<Ephemeris<InertialFrame> const*> ephemeris,
-    not_null<MassiveBody const*> primary,
-    not_null<MassiveBody const*> secondary)
+    BarycentricRotatingReferenceFrame(
+        not_null<Ephemeris<InertialFrame> const*> ephemeris,
+        not_null<MassiveBody const*> primary,
+        not_null<MassiveBody const*> secondary)
+    : BarycentricRotatingReferenceFrame(ephemeris,
+                                        primary,
+                                        std::vector{secondary}) {}
```

Comment:
Is this constructor used by anything now?

Reply:
At least the plugin and the test.